pages/public_cloud/public_cloud_network_services/configuration-04-update_subnet/guide.en-gb.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Update a subnet options
+excerpt: Find out how to update the options of an existing subnet
+updated: 2024-01-19
+---
+
+## Objective
+
+This page explains how to update the options of an existing subnet. This can be needed if you want to comply with the pre requisites to be able to use the Public Cloud Gateway : have a gateway IP defined in your subnet.
```

Comment:
```suggestion
This page explains how to update the properties of an existing subnet. This can be needed if you want to comply with the pre requisites to be able to use the Public Cloud Gateway : have a gateway IP defined in your subnet.
```